ui/pages/home/home.component.js

```diff
@@ -87,7 +87,10 @@ export default class Home extends PureComponent {
     forgottenPassword: PropTypes.bool,
     suggestedAssets: PropTypes.array,
     unconfirmedTransactionsCount: PropTypes.number,
-    shouldShowSeedPhraseReminder: PropTypes.bool.isRequired,
+    shouldShowSeedPhraseReminder: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.object,
+    ]).isRequired,
```

Comment:
I don't think I understand what happened here, why `shouldShowSeedPhraseReminder` could be an object now.

Reply:
Fixed